app/assets/stylesheets/variables.sass

```diff
@@ -0,0 +1,15 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+// Fonts
+
+$primary-font: 'Roboto Slab', serif;
+$secondary-font: 'Roboto', sans-serif;
+
+// Colors - named using http://chir.ag/projects/name-that-color/
+
+$ebony-clay: #252C41; // Main color
```

Comment:
It would be best to call these what they represent, instead of what they are. Think of variable names in Ruby; which is more useful to you: `main_color` or `ebony_clay`? Which are you most likely to remember?

Also if you wanted to change the primary color from this to something else, you would have to make a bunch of changes which is what the vars are supposed to ward against.